### DISCUSSION TITLE
How to configure dark mode defaults

### DISCUSSION THREAD

**Discussion by taylorhlabs** [tailwindlabs/tailwindcss#5679] [Help] [answered]
Is there a way to configure dark mode defaults in tailwind.config.js file? For instance, how could I set all links (`<a>` tags) to be white in dark mode?

**Comment by adamwathan** [comment, marked as answer]
Hey! The best way to do that is to just write some CSS:

```css
@layer base {
  a {
    @apply dark:text-white;
  }
}
```

Here's some relevant documentation: https://tailwindcss.com/docs/adding-base-styles#using-css

**Reply by taylorhlabs** [reply]
Brilliant. Thanks! Might be helpful to add your snippet to the Base Styles or Dark Mode docs page for folks like me that overlook this simple solution for setting dark mode base styles.
